pkg/provider/provider.go

```diff
@@ -170,6 +170,10 @@ func (k *kubeProvider) invalidateResources() {
 	k.resources = nil
 }
 
+func (k *kubeProvider) GetSchema(ctx context.Context, req *pulumirpc.GetSchemaRequest) (*pulumirpc.GetSchemaResponse, error) {
+	panic("implement me")
```

Comment:
Is panicing "right" here - or can this just return `Unimplemented` explicitly?

Reply:
Updated to return Unimplemented error